console/src/app/modules/policies/private-labeling-policy/private-labeling-policy.component.html

```diff
@@ -101,7 +101,7 @@ <h2>{{ 'POLICY.PRIVATELABELING.TITLE' | translate }}</h2>
   <div *ngIf="previewData && data" class="lab-policy-content">
     <mat-accordion class="settings">
       <mat-expansion-panel class="expansion">
-        <mat-expansion-panel-header [attr.data-e2e]="'policy-category'">
+        <mat-expansion-panel-header data-e2e]="policy-category">
```

Comment:
did this compile locally?

```suggestion
        <mat-expansion-panel-header data-e2e="policy-category">
```